TestCases/nicf/coolprop/coolprop_nozzle.cfg

```diff
@@ -1,123 +0,0 @@
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%                                                                              %
-% SU2 configuration file                                                       %
-% Case description: Non-ideal compressible fluid flow in a converging-         %
-%                   diverging supersonic nozzle with CoolProp library          %
```

Comment:
Keep the old case please, you already added a mesh and restart for it, so it's better to keep it at this point.

Reply:
Get it.
I keep the old case.